README.md

```diff
@@ -1,28 +1,105 @@
+    _(Parameters: path for your new database, language, driver source directory, build command.)_
+
+1. Analyze your CodeQL database:
+    
+    CodeQL's analysis output is provided in the form of a SARIF log file. For a human readable format, drop the SARIF file into [SARIF Viewer Website](https://microsoft.github.io/sarif-web-component/). (If there are violations, they will show up. If not, the page will not update.)
+    ```
+    D:\codeql-home\codeql>codeql database analyze <path to database> --format=sarifv2.1.0 --output=<"path to output file".sarif> <path to query/suite to run>
+    ```
+    Example: `codeql database analyze D:\DriverDatabase --format=sarifv2.1.0 --output=D:\DriverAnalysis1.sarif D:\codeql-home\Windows-driver-developer-supplemental-tools\src\suites\windows_driver_mustfix.qls`
+
+    _(Parameters: path to new database, format, output sarif file, path to CodeQL query or query suite to use in analysis.)_
+
+    **Note**: Be sure to check the path to the suite or query you want to run, not every branch has the same file structure.
+
+1. ***For WHCP Users Only***
+
+    Copy the SARIF log file to the parent directory of your driver file in order to convert it to DVL format. You can also modify your output location in the `codeql database analyze` step in order to skip this additional step. Continued instructions at [CodeQL and the Static Tools Logo Test, Driver Verification Log DVL Consumption of SARIF Output](https://learn.microsoft.com/en-us/windows-hardware/drivers/devtest/static-tools-and-codeql#driver-verification-log-dvl-consumption-of-sarif-output).
+    ```
+    D:\codeql-home\codeql>copy <path to SARIF output file> <path to driver directory>
+    ```
+    Example: `D:\codeql-home\codeql> copy D:\DriverAnalysis1.sarif D:\Drivers\SingleDriver`
+
+## Navigation
+
+Windows drivers queries are in the `src/drivers` directory.
+
+Non-driver Microsoft-specific queries provided by Microsoft are in the `src/microsoft` directory.
+
+Query suites are located in the `src/suites` directory and contain the Must-Fix and Recommended-Fix suites used by the WHCP Program.
 
 ## Contributing
+This project welcomes contributions, feedback, and suggestions!
+
+We are in the process of setting up pull request checks, but to ensure our requirements are met, please ensure the following are complete with your pull request:
+1. Pull request description contains a concise summary of what changes are being introduced.
+2. Only one query or query group are introduced per pull request.
+3. If there are changes to an existing query, increase the tracked version of that query.
+4. Write at least one positive and one negative unit test for any new queries or changed queries.
+5. Run all unit tests.
+6. Run `codeql database create` and `codeql database analyze` successfully on a valid driver before merging.
+7. Update the help menu associated with any new queries.
```

Comment:
What is the "help menu" in this context?

Reply:
Good catch. I mean the .qhelp files associated with the queries, if there is any change in functionality for the end user. I will clarify.